storey/redis_driver.py

```diff
@@ -194,7 +194,7 @@ def convert_redis_value_to_python_obj(cls, value):
             ret = math.inf
         elif value == "-inf":
             ret = -math.inf
-        elif value == "nan" or value == "-nan":
+        elif value == "nan" or value == "-nan" or value == "0/0":
```

Comment:
Any chance we will also get "-0/0"?

Reply:
Don't see how, but on the other hand Assaf added "-nan". I will add -0/0 as well to keep things symmetric.